index.html

```diff
@@ -18,7 +18,9 @@
 <h1>Hier de doos met slaapkamerspullen</h1>
 
 <h2>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Architecto assumenda autem distinctio, dolorum earum enim exercitationem facilis illo itaque magnam placeat qui quibusdam quos ratione saepe tenetur veritatis voluptatem voluptatum.
-<p><article>blablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablabl
+
+    <h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Delectus distinctio eveniet excepturi, inventore iure modi placeat quasi quis sed unde ut vitae voluptas? Consequuntur distinctio error inventore laboriosam nemo non officia porro velit voluptates? Accusantium ad adipisci aliquam amet autem beatae blanditiis cum deserunt dignissimos dolores in iusto nam nesciunt nisi non odit, officia omnis quasi qui repellendus, rerum sunt, tenetur voluptates! A accusamus adipisci autem cupiditate delectus dignissimos dolor, ea eos exercitationem facere id minima nemo nobis numquam odio pariatur repudiandae sequi sunt veritatis.</h3>
+    <p><article>blablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablabl
     <br>
     <br>ablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablabla
     <br>blablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablablabla</article></p>
```

Comment:
Hier is het ook beter om hier de tekt binnen aan p tag te plaatsen

Reply:
yes!